ha/monitor/k8s/object_monitor.py

```diff
@@ -167,3 +172,44 @@ def run(self):
                 break
         Log.info(f"Stopping the {self.name}...")
 
+    def _is_published_alert(self, alert) -> bool:
+        """
+        Check incoming alert is already published or not.
+        If incoming alert is not found in published alerts then,
+        it is a new alert to publish.
+        self._published_alerts = { resource_name: [alert1, alert2] }
+
+        Returns:
+            True if it is published already
+            False if it is a new alert
+        """
+        incoming_alert = alert.to_dict().copy()
+
+        # Alert with no status change also has new timestamp.
+        # So timestamp field should be ignored for validation.
+        if not incoming_alert.get('_timestamp') is not None:
```

Comment:
Can you clarify this check ?

Reply:
Even there is no change in status change, the watch stream repeats recent status in event at every 5 seconds with new timestamp. This is expected from watch stream. Rather than saying its an event, better can say its a response (as variable name defined for that in kubernetes watch stream module).

When no new event, but response is coming from stream, published and this new event is same and only the _timestamp differ. In this case I'm ignoring this _timestamp filed before the alert messages comparison.

Reply:
@ArchanaLimaye , correcting this comment as below may give more clarity. Fine?
```
        # Repeating alert also has new timestamp on same event.
        # So timestamp field should be ignored for validation.
```

Reply:
Done.